crates/loader/src/local.rs

```diff
@@ -416,7 +416,12 @@ impl LocalLoader {
                 continue;
             }
 
-            let app_root_path = src.strip_prefix(&self.app_root)?;
+            let app_root_path = match src.strip_prefix(&self.app_root) {
+                Ok(path) => path,
+                Err(error) => {
+                    bail!("Invalid file path outside application root, error: {error}")
```

Comment:
A couple of things on this:

1. There's probably not much point including the `StripPrefixError` in the output.  We're translating the "prefix not found" into a more meaningful error: we don't need to show how we know.
2. We need to say which file or pattern is causing the problem.  The user might have several entries in the `files` section, and would appreciate guidance to which one.  This may be a bit tricky to get in its exact form, because it looks like it's already been globbed by this point, but even the glob pattern would help them narrow down the problem.
3. Rather than saying "Invalid file path" (it may well be _valid,_ we just don't like it), consider phrasing it in terms of what the rules are and why this doesn't fall within them, e.g. `"/home/ivan/catpictures" cannot be mapped because it is outside the application directory. Files must be within the application directory.`

Reply:
Thanks a bunch for the detailed review. I have updated the commit accordingly. I have included the pattern for the user to identify where it's failing.